dashboard/app/controllers/admin_users_controller.rb

```diff
@@ -101,33 +101,59 @@ def manual_pass
     redirect_to :manual_pass_form
   end
 
+  # display the user search form, users found, and permissions for first user found
+  # grant / revoke permissions actions redirect back to this form to display the updated permission list
```

Comment:
Right now, this requires the `admin` privilege to access. Does this need to be accessible to non-admins?